### PR TITLE
1576(docs): documents config option

### PR DESCRIPTION
**Pull request by m4sterbunny**
Fixes https://github.com/hyperledger/besu-docs/issues/1576

### CODE REVIEW

**Comment by non-fungible-nelson** [comment]
LGTM

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+
+```bash
+use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+</Tabs>
+
+Enables fast startup from an existing genesis state hash. The default is `false`.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5720]:
```suggestion
Enables or disables fast startup from an existing genesis state hash. The default is `false`.
```

For consistency with other boolean options.

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+
+:::danger
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5722]:
```suggestion
:::warning
```

We usually don't use the [red danger](https://docusaurus.io/docs/3.0.1/markdown-features/admonitions) admonition except for the most critical security warnings.

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+
+As this method avoids validating the genesis state hash, it trades-off security for faster node startup times. Its use is only recommended if you are certain that you have not modified your genesis file or database and understand the security implications.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5724]:
```suggestion
Enabling this option avoids validating the genesis state hash, trading off security for faster node startup times. We only recommend using this option if you are certain that you have not modified your genesis file or database and understand the security implications.
```

**Review by alexandratran — COMMENTED** [review]
Some minor comments. Also please use the a [GitHub keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in your PR description (e.g. "Fixes #1576") so that when this PR is merged, the linked issue is automatically closed.

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+```bash
+--use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+use-cached-genesis-state-hash=true
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:5705]:
```suggestion
BESU_USE_CACHED_GENESIS_STATE_HASH=true
```

**Review by macfarla — COMMENTED** [review]
the option name is 
--genesis-state-hash-cache-enabled

see https://github.com/hyperledger/besu/pull/6758/files

File: docs/public-networks/reference/cli/options.md [hunk 5]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
 The default file name is `txpool.dump` in the [data directory](#data-path).
 
+### `use-cached-genesis-state-hash`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--use-cached-genesis-state-hash=[=<true|false>]
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5689]:
```suggestion
--genesis-state-hash-cache-enabled=[=<true|false>]
```

File: docs/public-networks/reference/cli/options.md [hunk 6]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--use-cached-genesis-state-hash=true
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5697]:
```suggestion
--genesis-state-hash-cache-enabled=true
```

File: docs/public-networks/reference/cli/options.md (hunk 4 above)
Comment (alexandratran) [reply docs/public-networks/reference/cli/options.md:5705]:
```suggestion
BESU_GENESIS_STATE_HASH_CACHE_ENABLED=true
```

File: docs/public-networks/reference/cli/options.md [hunk 7]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
...
+```bash
+use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+<TabItem value="Example configuration file" label="Example configuration file"> 
+
+```bash
+use-cached-genesis-state-hash=true
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5713]:
```suggestion
genesis-state-hash-cache-enabled=true
```

File: docs/public-networks/reference/cli/options.md [hunk 8]
```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
 The file is created on shutdown and reloaded during startup.
 The default file name is `txpool.dump` in the [data directory](#data-path).
 
+### `use-cached-genesis-state-hash`
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:5682]:
```suggestion
### `genesis-state-hash-cache-enabled`
```

And re-order to be in alphabetical order.

**Review by alexandratran — APPROVED** [review]
LGTM

**Pull request merged**